routes/api.php

```diff
@@ -85,4 +85,12 @@
         Route::get('{id}/groups', 'API\NetworkController@getNetworkGroupsv2');
         Route::get('{id}/events', 'API\NetworkController@getNetworkEventsv2');
     });
+
+    Route::prefix('/moderate')->group(function() {
```

Comment:
Genuine question to which I don't know the answer - is this the most logical/RESTful naming for these resources?

so we will be calling something like

`api/v2/moderate/groups` or ``api/v2/moderate/events`

but should it instead be based around the resources in question?  i.e. groups and events.  e.g. it could be 

`api/v2/groups?requires_moderation=true`
`api/v2/events?requires_moderation=true`

I do not really know, and certainly won't lose sleep over it, but would be interested in your view, as we said we try to keep our v2 api naming tidy.